rebar.config.lock

```diff
@@ -12,37 +13,37 @@
                    "ad400896af5b1ad8b4f7a4d34e609b5a990640bb"}},
        {rebar_lock_deps_plugin,".*",
                                {git,"git://github.com/seth/rebar_lock_deps_plugin.git",
-                                    "9711549b8a84b065eb2edc22f8eb6ff85e3c94e8"}},
+                                    "7a5835029c42b8138325405237ea7e8516a84800"}},
```

Comment:
We bump a lot of deps here. Which ones do we *need* to bump to get this building on erl 17? My concern here is that with the lack of unit testing, we're more likely to run into some issues here by bumping all these deps to latest.